operator/cmd/mesh/manifest_shared_test.go

```diff
@@ -71,7 +72,7 @@ var (
 	testClient            client.Client
 	testReconcileOperator *istiocontrolplane.ReconcileIstioOperator
 
-	allNamespacedGVKs = append(helmreconciler.NamespacedResources,
+	allNamespacedGVKs = append(helmreconciler.NamespacedResources(&version.Info{Major: "1", Minor: "25"}),
```

Comment:
LGTM, except here, if minor version `25` can be replaced with other one, could we defined a constant for it, including the change at L#209 in this file ?